docs/src/components/code-block/actions.tsx

```diff
@@ -6,7 +6,7 @@ import { CopyToClipboard } from 'react-copy-to-clipboard'
 
 import pkg from '../../../package.json'
 import usePrevious from '../../hooks/use-previous'
-import styles from './styles.module.css'
+import * as styles from './styles.module.css'
```

Comment:
How come we had to make this a wildcard?

Reply:
I was getting errors that `styles` was undefined.  The wildcard syntax was what I found in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/styling/css-modules/#css-module-example).

Reply:
I can look into it more - maybe we can go back

Reply:
If it's in the docs, then don't worry about it. It's just an anti-pattern for imports...but...gatsby be gatsby... 🙈